### PR TITLE
changes in readme file for build on windows os

### PR DESCRIPTION
**Pull request by marahmbawwab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,18 @@ You must install [Msys2](https://www.msys2.org/) first, then **remember choosing
 
 The **default MSYS environment** is based on Cygwin, which **won't work**!
 
+*steps in order to build for windows:
+-Download msys2 then open the shell and type those commands respectively:
```
Comment (pgervais) [inline README.md:83]: For this list to render correctly, you need to add a space after each dash. Example:

```
- Download msys2 then ...
- Open msys2-mingw64 shell from ...
```

You can check the rendering of a markdown (.md) file on Github by clicking the three-dot menu on the right and choosing "view file".
```

File: README.md [hunk 2]
```diff
@@ -79,6 +79,18 @@ You must install [Msys2](https://www.msys2.org/) first, then **remember choosing
...
+ update pacman -Syu then pacman -su.
```
Comment (pgervais) [inline README.md:84]: Using backticks here will help the reader understand what exactly is to be run. See the difference:
Did you mean to run `update pacman -Syu` or `pacman -Syu`?

This was formatted by writing

```
Did you mean to run `update pacman -Syu` or `pacman -Syu`?
```

File: README.md [hunk 3]
```diff
@@ -79,6 +79,18 @@ You must install [Msys2](https://www.msys2.org/) first, then **remember choosing
...
+-Open msys2-mingw64 shell from the start menu.
+-Type these three commands: 
+ pacman -s  
+ pacman --needed -s git mingw-w64-x86-gcc base-devel
+ pacman mingw-w64-x86-toolchain
```
Comment (pgervais) [inline README.md:89]: You can use this technique to make those three lines of code look nicer: https://www.markdownguide.org/basic-syntax/#code-blocks-1

(TL;DR: indent those lines with 8 spaces)

**Review by pgervais — CHANGES_REQUESTED** [review]
Thanks for  the detailed instructions! There are a few minor issues with the Markdown formatting, which makes the text very hard to read once rendered. See my comments on the files.

File: README.md [hunk 4]
```diff
@@ -79,6 +79,28 @@ You must install [Msys2](https://www.msys2.org/) first, then **remember choosing
...
+-  run `pacman --needed -s git mingw-w64-x86-gcc base-devel`.
+
+-  run `pacman mingw-w64-x86-toolchain`.
+
+-  Add path to bin directory for both msys64 and mingw64 to the system enviroment variables.
+
+   For msys64: `..\msys64\mingw64\bin` 
+   For mingw64 `...\msys64\usr\bin` 
+
+-  Continue the commands below:
```
Comment (pgervais) [inline README.md:102]: `Continue with the commands below:`

**Comment by fundamental** [comment]
Thanks for the documentation enhancement :+1:

**Pull request merged**